config/locales/zh-TW.yml

```
@@ -0,0 +1,36 @@
+zh-TW:
+  activerecord:
+    models:
+      mission: Mission
+    attributes:
+      mission:
+        title: 標題
+        content: 內容
+        started_at: 開始時間
+        ended_at: 結束時間
+        submit: 送出
+  missions:
+    index:
+      title: 任務列表
+      link:
+        create: 新增任務
+        edit: 修改
+        delete: 刪除
+      notice:
+        confirm: 確認刪除?
```

Nit:
從前端要使用到的地方來說，也許命名為 `alert.deletion` 會更合適？
```erb
<%# current %>
<%= link_to t('.link.delete'), ..., data: { confirm: t('.notice.confirm') } %>

<%# proposed %>
<%= link_to t('.link.delete'), ..., data: { confirm: t('.alert.deletion') } %>
```
```suggestion
      alert:
        deletion: 確認刪除？
```